src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -157,6 +157,161 @@ private[spark] object Config {
   private def k8sLabelListValidator(labelList: String): Boolean = {
     parseCommaSeparatedK8sLabels(labelList).forall(_.isSuccess)
   }
+
+  val ARMADA_QUEUE: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.queue")
+      .doc("The Armada queue that will be used for running the driver and executor pods.")
+      .version("1.0.0")
+      .stringConf
+      .createWithDefault("default")
+
+  val DRIVER_JOB_SET_ID: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.driver.job.set.id")
```

Comment:
Spark config tries to minimize namespace levels:
```suggestion
    ConfigBuilder("spark.armada.driver.jobset")
```